storage/sqlite/sqlite.go

```diff
@@ -51,6 +51,49 @@ func (r *Backend) Open(options url.Values) {
 	r.tenant = make(map[string]*sql.DB)
 }
 
+func (r Backend) GetLastDataItems(tenant string, id string, numOfItems int) ([]storage.DataItem, error) {
+	res := make([]storage.DataItem, numOfItems)
+	db, _ := r.GetTenant(tenant)
+	var count int
+	// check if id exist
+	if !r.IdExist(tenant, id) {
+		return nil, fmt.Errorf("Metric does not exist in database.")
+	}
+
+	countQuery := fmt.Sprintf("select count(timestamp) from %s", id)
+	db.QueryRow(countQuery).Scan(&count)
+
+	if count < numOfItems {
+		return nil, fmt.Errorf("Not Enough items in database: Found %d, Queried: %d", count, numOfItems)
+	}
+	sqlStmt := fmt.Sprintf(`select timestamp, value
+	from '%s'
+	order by -timestamp %s limit %d`, id, numOfItems)
+	rows, err := db.Query(sqlStmt)
+	if err != nil {
```

Comment:
No need to handle the error here, let the caller handle it, much simpler and easy to test 😄

Reply:
👍 Cool